### PR TITLE
Demultiplexing scRNA data hands-on material

### PR DESCRIPTION
**Pull request by mtekman**

### CODE REVIEW

File: topics/transcriptomics/tutorials/scrna-umis/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,553 @@
...
+ ![FastQC Plot](../../images/scrna_barcodes_fastq.png "FastQC plot of Forward read")
+
+
+Here we can see the three distinct regions along the x-axis that correspond to our expected [CEL-Seq2 Schema](#the-cel-seq2-protocol):
+
+ * 01-06bp: smooth, relatively constant bases.
+ * 07-12bp: noisy, highly varied distribution of bases.
+ * 13-30bp: T-dominated region
+
+We can see that the distribution of the first 6bp is relatively more even than the following 6bp which seems to have more extreme variation.
```
Comment (bgruening) [inline topics/transcriptomics/tutorials/scrna-umis/tutorial.md]: This sentence sounds wired and it less clear imho.

File: topics/transcriptomics/tutorials/scrna-umis/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,553 @@
...
+> >  * `@J00182:75:HTKJNBBXX:2:2222:13301:35690`
+> >  * `@J00182:75:HTKJNBBXX:2:1114:12469:11073`
+> >  all have the cell barcode `GGTAAC`.
+> >
+> > 2. Reads:
+> >  * `@J00182:75:HTKJNBBXX:2:1203:25022:13763`
+> >  * `@J00182:75:HTKJNBBXX:2:2222:13301:35690`
+> >  are duplicates, since they both have the same cell barcode and same UMI.
+> >
+> > However, they are not direct duplicates of each other. since their sequences are different. However, they do stem from the same read as evidenced by their overlap:
```
Comment (bgruening) [inline topics/transcriptomics/tutorials/scrna-umis/tutorial.md]: other, since?

Comment (bgruening) [reply topics/transcriptomics/tutorials/scrna-umis/tutorial.md]: What does this mean if you don't have a direct duplicate? Does it get filtered? Does it matter?

File: topics/transcriptomics/tutorials/scrna-umis/tutorial.md [hunk 3]
```diff
@@ -0,0 +1,553 @@
...
+questions:
+  - "What are barcodes?"
+  - "What is their purpose?"
+  - "How are barcodes formatted?"
+objectives:
+  - "Demultiplex FASTQ data via UMI-tools"
+  - "Understand and validate the extraction of barcodes"
+time_estimation: "2h"
+key_points:
+  - "Verifying the distribution of barcodes via a FASTQC plot"
```
Comment (bgruening) [inline topics/transcriptomics/tutorials/scrna-umis/tutorial.md]: Its a key point ... so maybe `Verify the ....`

File: topics/transcriptomics/tutorials/scrna-umis/tutorial.md [hunk 4]
```diff
@@ -0,0 +1,553 @@
...
+> |  | UMIs in Cell 1 |
+> |--|----------------|
+> | Gene Red | 2 |
+> | Gene Blue | 2 |
+{: .matrix}
+
+
+> ### {% icon question %} Questions about UMIs
+> 
+> 1. Are UMIs specific to genes? i.e. Can the same UMI map to different genes?
```
Comment (bgruening) [inline topics/transcriptomics/tutorials/scrna-umis/tutorial.md]: Isn't the answer to 1.Q1 no and to 1.Q2 yes.? i.e. in the beginning of a sentence is also unusal.

**Comment by mtekman** [comment]
made the changes, thanks!

**Pull request merged**